File: src/ois_tournament3/tournament.pas
{ NOTE: it is recommended to use this even if you don't understand the following code }

const
    MAXN = 131072;

var
    N, i, runnerup, winner : LongInt;
    P                      : Array[0..MAXN-1] of LongInt;

begin
{
    uncomment the two following lines if you want to read/write from files
    assign(input,  'input.txt');  reset(input);
    assign(output, 'output.txt'); rewrite(output);
}

    ReadLn(N);

    for i:=0 to N-1 do
        Read(P[i]);
    ReadLn();

    winner := 0;
    runnerup := 0;

    { INSERT YOUR CODE HERE }


    WriteLn(winner, ' ', runnerup);

end.

File: src/ois_tournament3/tournament.java
// NOTE: it is recommended to use this even if you don't understand the following code.

import java.util.*;
import java.io.*;
import java.lang.*;

public class tournament {

    public static void main(String[] args) throws FileNotFoundException, IOException {
        Locale.setDefault(Locale.US);
        InputStream fin = System.in;
        OutputStream fout = System.out;
        // uncomment the two following lines if you want to read/write from files
        // fin = new FileInputStream("input.txt");
        // fout = new FileOutputStream("output.txt");

        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(fout));
        reader = new BufferedReader(new InputStreamReader(fin));
        scn = new StringTokenizer(reader.readLine());

        int N = Integer.parseInt(next());

        int[] P = new int[N];
        for (int i = 0; i < N; ++i)
            P[i] = Integer.parseInt(next());

        int winner = 0, runnerup = 0;


        // INSERT YOUR CODE HERE


        writer.write(String.valueOf(winner));
        writer.write(' ');
        writer.write(String.valueOf(runnerup));
        writer.write('\n');

        writer.flush();
    }

    static String next() throws IOException {
        while (!scn.hasMoreTokens()) scn = new StringTokenizer(reader.readLine());
        return scn.nextToken();
    }

    static BufferedReader reader;
    static StringTokenizer scn;
}
